crates/oxc_linter/src/snapshots/no_dangerously_set_inner_html.snap

```diff
@@ -0,0 +1,33 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_dangerously_set_inner_html
+---
+  ⚠ eslint-plugin-react(no-danger): Do not use `dangerouslySetInnerHTML` prop
+   ╭─[no_dangerously_set_inner_html.tsx:1:1]
+ 1 │ <div dangerouslySetInnerHTML={{ __html: "" }}></div>;
+   ·      ────────────────────────────────────────
```

Comment:
Shall we shorten then span to `dangerouslySetInnerHTML` only? The rule of thumb is point the user to the shortest span where the error is.